api/app/Actions/EventType/ChangeDisabledByIdAction.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Actions\EventType;
+
+use App\Entity\EventType;
+
+final class ChangeDisabledByIdAction
+{
+    public function execute(ChangeDisabledByIdRequest $request): void
+    {
+        $eventType = EventType::find($request->getId());
+        $eventType->disabled = $request->getDisabled();
+        $eventType->save();
```

Comment:
@aLLeXUs please check, I suppose here must've been EventTypeRepository, mustn't it?

Reply:
Indeed